### ISSUE TITLE
Show share of controllable vs non-controllable energy for production and capacity

### ISSUE THREAD

**Issue by Styrnikov** [electricitymaps/electricitymaps-contrib#3722]
Could it be possible to represent synthetically the share of controlable power (thermal, hydro, géo thermal, bio fuels, storage) and non or poorly controlable (wind, PV)? Both in installed and delivered capacity.

**Comment by jarek** [comment]
Thank you for the suggestion! I renamed it so it's a little easier to glance at in the issues list.

Hydro is a potentially interesting case - run-of-river hydro isn't entirely dispatchable, as while in most cases plants are able to choose to produce energy, this might not be an option if water is low (e.g. due to dry season). Though in any case Electricity Map doesn't split out run-of-river and reservoir hydro.

**Comment by jarek** [comment]
Sorry, @Styrnikov, I might need to ask for clarification. I renamed the issue to refer to "dispatchable generation", but this might not have been correct. As pointed out in Wikipedia article https://en.wikipedia.org/wiki/Dispatchable_generation, most coal and nuclear (not all, but most) aren't really useful for filling in for wind and solar because their time to increase production is several hours. Are you interested in precisely "controllable" (thus including a lignite coal plant that takes a day to spin up), or in "dispatchable" in the sense of "useful for compensating for variable renewable production"?

**Comment by Styrnikov** [comment]
I agree the dispatchability of run of river hydro is disputable, since it’s
fully conditionned by the availability of water. However, when you manage
wisely a valley, you know what will be the output of a run of river unit
when you turbine water upstream. There is no equivalent for wind or solar.

As regards coal and nuclear one must distinguish the base load mode they
are operated in from their lack of or reduced dispatchability,
notwithstanding their maneuverability constraints.
Whatever notice time you give to a PV or wind power unit won’t help you to
increase its output.
And thats a huge difference.

**Comment by jarek** [comment]
I renamed the issue to controllable since it sounds like you're interested in distinguishing specifically the non-controllable sources.

**Comment by VIKTORVAV99** [comment]
We are closing down some longstanding issues that have't had any activity in a while. If you believe this is still an issue let us know and we can reopen it.
